packages/mpc-tls/notary/src/main.rs

```diff
@@ -0,0 +1,25 @@
+use eyre::{eyre, Result};
+use structopt::StructOpt;
+use tracing::debug;
+
+use notary_server::{
+    init_tracing, parse_config_file, run_server, CliFields, NotaryServerError,
+    NotaryServerProperties,
+};
+
+#[tokio::main]
+async fn main() -> Result<(), NotaryServerError> {
+    // Load command line arguments which contains the config file location
+    let cli_fields: CliFields = CliFields::from_args();
+    let config: NotaryServerProperties = parse_config_file(&cli_fields.config_file)?;
+
+    // Set up tracing for logging
+    init_tracing(&config).map_err(|err| eyre!("Failed to set up tracing: {err}"))?;
+
+    debug!(?config, "Server config loaded");
+
+    // Run the server
+    run_server(&config).await?;
```

Comment:
Is `PORT` managed the same way here as it in the `prover`?

Reply:
yes it is, you can take a look at the start scripts for the notary and prover for how to pass data into it.